app/views/developments/_form.html.haml

```diff
@@ -12,4 +12,4 @@
 
 - if @development.audit_changes?
   .govuk-form-group
-    = f.input :audit_comment, label: 'Changelog'
+    = f.input :audit_comment, label: 'Reason for changes to legal agreement'
```

Comment:
Should we add `, required: true` to this too?

Reply:
I wondered that — perhaps it belongs to https://trello.com/c/g1oNFIu3/59-editing-a-development-required-form-hint ?

Reply:
ah yes